core/arch/disassemble_shared.c

```diff
@@ -159,6 +159,12 @@ dsts_first(void)
     return TESTANY(DR_DISASM_INTEL | DR_DISASM_ARM, DYNAMO_OPTION(disasm_mask));
 }
 
+static inline bool
```

Comment:
> k1 src0 src2 -> dst

s/src2/src1/

nit: k1 is actually src0 according to instr_get_src(instr, 0) which makes this a little confusing

Reply:
Done